.travis.yml

```diff
@@ -7,7 +7,6 @@ env:
     - SCRIPT=lint
     - SCRIPT=tests
 python:
-  - "3.5"
```

Comment:
Why did you remove 3.5?

Reply:
* We no longer have up to date packages for 3.5 on conda-forge
* We don't use Python 3.5 anymore, so no reason to keep supporting it on our side

Our idea was to "keep supporting until it doesn't cost us anything to do so", which is no longer the case really. 🤷‍♂